tdrs-frontend/src/components/SiteMap/SiteMap.test.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import SiteMap from './SiteMap'
+import { mount } from 'enzyme'
```

Comment:
For these two testing frameworks / js utilities, we are using them to do the same thing. I am under the assumption that we would use one or the other rather than both.

Reply:
@msohanifr and I discussed this and we agree, there's usage of both libraries throughout the tests but we should really standardize on one. we're going to make another work item for it

Reply:
https://github.com/raft-tech/TANF-app/issues/2196